.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: tt
+# Tests runs different tests (test_abci_apps, test_abci_cli, test_apps)
+# This workflow runs on every push to master or release branch and  every pull requests
+# All jobs will pass without running if no *{.go, .mod, .sum} files have been modified
+on:
+  pull_request:
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.15"
+      - uses: actions/checkout@v2
+        with: 
+          fetch-depth: 0
+      - run: |
+         git branch -a | grep '^v0.*.x' | tac | head -2 > output
```

Comment:
For now, the only release branch that supports e2e tests is v0.34.x - they weren't written for <= v0.33